### ISSUE TITLE
Solona

### ISSUE THREAD

**Issue by 4favorapp** [openware/opendax#357]
Is it possible to add Solona to Opendax ?

**Comment by kanakan34** [comment]
> Is it possible to add Solona to Opendax ?

yes possible. you can add any cryptocurrency to opendax. you need add gem plugin to peatio before.
